### PR TITLE
Fix incorrect test setup where Package did not have a Version

### PR DESCRIPTION
**Pull request by xavierdecoster**

### CODE REVIEW

File: tests/NuGetGallery.Facts/Controllers/PackagesControllerFacts.cs [hunk 1]
```diff
@@ -399,8 +399,8 @@ public class TheDisplayPackageMethod
                             Id = "Foo",
                             Owners = new List<User>()
                         },
-                        Version = "2.0.0a",
-                        NormalizedVersion = "2.0.0a",
+                        Version = "2.0.0",
```
Comment (skofman1) [inline tests/NuGetGallery.Facts/Controllers/PackagesControllerFacts.cs]: why change this?

File: tests/NuGetGallery.Facts/Controllers/CuratedFeedsControllerFacts.cs [hunk 2]
```diff
@@ -189,14 +189,15 @@ public class TheListPackagesAction
                     Id = "RedPill",
                     Key = 2,
                     DownloadCount = 0,
-                    Packages = new []
+                    Packages = new[]
```
Comment (loic-sharma) [inline tests/NuGetGallery.Facts/Controllers/CuratedFeedsControllerFacts.cs]: Why remove the space between the `new` and the `[]`? `[]` is essentially a type...

File: tests/NuGetGallery.Facts/Controllers/PackagesControllerFacts.cs (hunk 1 above)
Comment (xavierdecoster) [reply tests/NuGetGallery.Facts/Controllers/PackagesControllerFacts.cs]: It's an invalid version :)

File: tests/NuGetGallery.Facts/Controllers/CuratedFeedsControllerFacts.cs (hunk 2 above)
Comment (xavierdecoster) [reply tests/NuGetGallery.Facts/Controllers/CuratedFeedsControllerFacts.cs]: auto-formatting of doc in VS

**Pull request merged**